user/git-extras/template.py

```diff
@@ -1,13 +1,47 @@
 pkgname = "git-extras"
-pkgver = "7.1.0"
+pkgver = "7.2.0"
 pkgrel = 0
 depends = ["git", "bash"]
 pkgdesc = "Extra Git utilities"
 maintainer = "ttyyls <contact@behri.org>"
 license = "MIT"
 url = "https://github.com/tj/git-extras"
 source = f"{url}/archive/refs/tags/{pkgver}.tar.gz"
-sha256 = "e5c855361d2f1ec1be6ee601247153d9c8c04a221949b6ec3903b32fa736f542"
+sha256 = "f570f19b9e3407e909cb98d0536c6e0b54987404a0a053903a54b81680c347f1"
+
+
+def do_build(self):
+    for cmd in list(self.find("bin", "git-*", files=True)):
+        if cmd in [
+            "bin/git-authors",
+            "bin/git-changelog",
+            "bin/git-commits-since",
+            "bin/git-count",
+            "bin/git-effort",
+            "bin/git-local-commits",
+            "bin/git-reauthor",
+            "bin/git-stamp",
+            "bin/git-summary",
+            "bin/git-undo",
+        ]:
+            with open(self.cwd / f"bin/{cmd}", "a") as outf:
+                self.do("cat", "helper/has-git-commit", stdout=outf)
+
+        if cmd in [
+            "bin/git-alias",
+            "bin/git-bulk",
+            "bin/git-extras",
+            "bin/git-force-clone",
+            "bin/git-fork",
+            "bin/git-get",
+            "bin/git-ignore",
+            "bin/git-setup",
+            "bin/git-standup",
+        ]:
+            pass
```

Comment:
this whole thing acts a subtitute to https://github.com/tj/git-extras/blob/main/Makefile#L68

Reply:
why not just the makefile install for the entire thing again

Reply:
needs a patch.. which i'll add instead of trying to recreate it in the template.